avocado/core/runner.py

```diff
@@ -447,6 +447,7 @@ def sigtstp_handler(signum, frame):     # pylint: disable=W0613
                                            ignore_window)
                         stage_1_msg_displayed = True
                     ignore_time_started = time.time()
+                    os.kill(proc.pid, signal.SIGINT)
```

Comment:
That's not needed. We received a SIGINT and the subprocess should just receive the same signal and handle it properly, just like it receives when we hit Ctrl+C. Problem is with the handler in the subprocess... sending a SIGTERM here will just hide the issue.